docs/zero-to-hero/03-intermediate/04-cross-contract-calls.md

```diff
@@ -169,86 +170,59 @@ pub fn claim_reward_new_account(
     new_pk: PublicKey,
     memo: String,
 ) -> Promise {
-    // Logic we'll skip
-    ext_linkdrop::create_account(
-        new_acc_id.parse().unwrap(),
-        new_pk,
-        AccountId::from(self.creator_account.clone()),
-        reward_amount,
-        GAS_FOR_ACCOUNT_CREATION,
-    )
-    .then(
-        // Chain a promise callback to ourselves
-        ext_self::callback_after_create_account(
-            crossword_pk, // First parameter
-            new_acc_id, // Second parameter
-            memo, // Third parameter
-            env::signer_account_pk(), // Fourth parameter
-            env::current_account_id(), // Which contract? "me"
-            0, // Deposit (in yoctoNEAR)
-            GAS_FOR_ACCOUNT_CALLBACK, // Gas
-        ),
-    )
+    ...
+    ext_linkdrop::ext(AccountId::from(self.creator_account.clone()))
+        .with_attached_deposit(reward_amount)
+        .with_static_gas(GAS_FOR_ACCOUNT_CREATION) // This amount of gas will be split
+        .create_account(new_acc_id.parse().unwrap(), new_pk)
+        .then(
+            // Chain a promise callback to ourselves
+            Self::ext(env::current_account_id())
+                .with_static_gas(GAS_FOR_ACCOUNT_CALLBACK)
+                .callback_after_create_account(
+                    crossword_pk,
+                    new_acc_id,
+                    memo,
+                    env::signer_account_pk(),
+                ),
+        )
 }
 ```
 
 Then the callback:
 
 ```rust
```

Comment:
was this meant to be a rust reference?

This is just a link in a code block

Reply:
good catch!